libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -1218,32 +1350,37 @@ public ClientManager getClientManager(String jwt, String url) {
     }
 
     /**
-     * Show dev support dialog
+     * Displays developer support for a specified Android activity.
+     *
+     * @param frontActivity The Android activity to chose developer support
+     *                      features for and display the dialog
      */
     public void showDevSupportDialog(final Activity frontActivity) {
         if (!isDevSupportEnabled()) {
             return;
         }
 
-        frontActivity.runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-                final LinkedHashMap<String, DevActionHandler> devActions = getDevActions(frontActivity);
-                final DevActionHandler[] devActionHandlers = devActions.values().toArray(new DevActionHandler[0]);
-
-                devActionsDialog =
-                        new AlertDialog.Builder(frontActivity)
-                                .setItems(
-                                        devActions.keySet().toArray(new String[0]),
-                                        (dialog, which) -> {
-                                            devActionHandlers[which].onSelected();
-                                            devActionsDialog = null;
-                                        })
-                                .setOnCancelListener(dialog -> devActionsDialog = null)
-                                .setTitle(R.string.sf__dev_support_title)
-                                .create();
-                devActionsDialog.show();
-            }
+        if (devActionsDialog != null) devActionsDialog.dismiss();
```

Comment:
This is the only new line in this block, which simply guards against showing more than one dialog at a time.  The rest is just a lambda conversion and whitespace.